.github/workflows/contracts.yml

```diff
@@ -148,6 +148,29 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npm publish --access=public --tag ${{ github.event.inputs.environment }} --network=${{ github.event.inputs.environment }}
 
+      # If we don't remove the `tbtc` contracts from `node-modules`, the
+      # `etherscan-verify` plugins tries to verify them, which is not desired.
+      - name: Prepare for verification on Etherscan
+        run: rm -rf ./node_modules/@keep-network/tbtc
+
+      - name: Verify contracts on Etherscan
```

Comment:
What if we extract `Verify contracts on Etherscan` to a separate job that would depend on `contracts-deployment-testnet` and use upload-artifacts/download-artifacts action to transfer data required for publication? With this apporach, we could avoid using annotations.

Reply:
Moved, 669c4d003f5c6b5aabe1ea730dd1342105fff24d.